components/scream/scripts/test-all-scream

```diff
@@ -122,6 +122,9 @@ OR
     parser.add_argument("--ctest-parallel-level", action="store", type=int, default=0,
         help="Force to use this value for CTEST_PARALLEL_LEVEL. If not provided, use default for given machine.")
 
+    parser.add_argument("-x", "--extra-verbose", action="store_true",
```

Comment:
Probably personal: any reason not to use the common `-v --verbose` flag, like most programs? "extra" makes me think we also have an option for "normal verbose", and "not verbose"...